File: app/SantanderInvestimento/app/src/main/java/lzacheu/com/br/santanderinvestimento/data/remote/contact/ContactResponse.java
package lzacheu.com.br.santanderinvestimento.data.remote.contact;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import lzacheu.com.br.santanderinvestimento.model.contact.InputField;

/**
 * Created by luiszacheu on 6/16/18.
 */

public class ContactResponse {

    @SerializedName("cells")
    private List<InputField> inputFields;

    public List<InputField> getInputFields() {
        return inputFields;
    }

    public void setInputFields(List<InputField> inputFields) {
        this.inputFields = inputFields;
    }
}

File: SantanderInvestimento/app/src/main/java/br/com/santander/santanderinvestimento/contact/data/entity/ContactResponse.kt
package br.com.santander.santanderinvestimento.contact.data.entity

import br.com.santander.santanderinvestimento.util.JSONConvertable
import com.google.gson.annotations.SerializedName

class ContactResponse(@SerializedName("cells") var cells: List<Contact>) : JSONConvertable